echopype/calibrate/api.py

```diff
@@ -34,11 +58,159 @@ def _compute_cal(
         return cal_obj.compute_Sp(waveform_mode=waveform_mode, encode_mode=encode_mode)
 
 
-def compute_Sv(echodata: EchoData, **kwargs):
-    """Compute volume backscattering strength (Sv) from raw data."""
+def compute_Sv(echodata: EchoData, **kwargs) -> xr.Dataset:
+    """
+    Compute volume backscattering strength (Sv) from raw data.
+
+    The calibration routine varies depending on the sonar type.
+    Currently this operation is supported for the following ``sonar_model``:
+    EK60, AZFP, EK80 (see Notes below for detail).
+
+    Parameters
+    ----------
+    echodata : EchoData
+        An `EchoData` object created by using `open_raw` or `open_converted`
+
+    env_params : dict
+        Environmental parameters needed for calibration.
+        Users can supply `"sound speed"` and `"absorption"` directly,
+        or specify other variables that can be used to compute them,
+        including `"temperature"`, `"salinity"`, and `"pressure"`.
+
+        For EK60 and EK80 echosounders, by default echopype uses
+        environmental variables stored in the data files.
+        For AZFP echosounder, all environmental parameters need to be supplied.
+        AZFP echosounders typically are equipped with an internal temperature
+        sensor, and some are equipped with a pressure sensor, but automatically
+        using these pressure data is not currently supported.
+
+    cal_params : dict
+        Intrument-dependent calibration parameters.
+
+        - for EK60 echosounder, allowed parameters include:
+          `"sa_correction"`, `"gain_correction"`, `"equivalent_beam_angle"`
+        - for AZFP echosounder, allowed parameters include:
+          `"EL"`, `"DS"`, `"TVR"`, `"VTX"`, `"equivalent_beam_angle"`, `"Sv_offset"`
+
+        Passing in calibration parameters for other echosounders
+        are not currently supported
+
+    waveform_mode : {"CW", "BB"}, optional
+        Type of transmit waveform.
+        Required only for data from the EK80 echosounder.
```

Comment:
Is waveform_mode *used* at all for other echosounders? That is, is it a parameter that only applies to EK80? If that's what you mean, the wording maybe you should be changed to, say: "Required for the EK80 echosounder; not used with any other echosounder"

Reply:
Yes, this is only required for EK80. The same is true for encode_mode also. I've changed both.